### DISCUSSION TITLE
Email wallets and third-party cookies

### DISCUSSION THREAD

**Discussion by jplsaez** [WalletConnect/walletconnect-monorepo#4497] [Developer Support]
Hello.

I realized that after adding "emailConnector" to my wagmi config to enable email wallets feature, lots of requests are continously sent from the user browser, most of them to "secure.walletconnect.com" and "auth.magic.link". 

Chrome browser complaints with lots of warnings about "Third-party cookie will be blocked. Learn more in the Issues tab."

In the issues tab the following message is shown:

"Reading cookie in cross-site context will be blocked in future Chrome versions
Cookies with the SameSite=None; Secure and not Partitioned attributes that operate in cross-site contexts are third-party cookies. In future Chrome versions, reading third-party cookies will be blocked. This behavior protects user data from cross-site tracking.
Please refer to the article linked to learn more about preparing your site to avoid potential breakage."

Two points:

 - I guess third party cookies should be avoided.
 - Is there any way to prevent those requests to keep flowing forever?

Best regards.

  Juan Pedro López

**Comment by glitch-txs** [comment]
hi, could you open an issue in the Web3Modal repo? 👍 thanks!

**Comment by jplsaez** [comment]
Hi Glitch,

I have just opened the same issue in "web3modal-sdk-support" repo.

Is it right?

Best regards.

   Juan Pedro López
